docs/sources/shared/datasources/tempo-editor-traceql.md

```diff
@@ -116,6 +130,11 @@ For more information on span details, refer to [Traces in Explore](https://grafa
 
 ![Selecting a trace ID or a span to view span details](/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-span-details-v11.png)
 
+Querying spansets with a large number of spans can negatively impact performance.
+You can use the **Span Limit** field in **Options** section of the TraceQL query editor.
+This field sets the maximum number of spans to return for each span set.
+If the **Span Limit** is set to `0`, then the capability is disabled.
```

Comment:
@carles-grafana With https://github.com/grafana/tempo/pull/4383, I understand that there will be a maximum value that can be set for the span limit.

Then:
* This default for Tempo OSS is 100: are we using this one for cloud also?
* If span limit > maximum, then the query returns an error, correct?
* Is there a limit when **Span Limit** is set to `0`?